solr/solr-ref-guide/modules/query-guide/pages/highlighting.adoc

```diff
@@ -509,6 +509,20 @@ Furthermore, phrases will be highlighted as a whole instead of word by word.  Cu
 +
 If either `hl.usePhraseHighlighter` or `hl.multiTermQuery` are set to `false`, then this setting is effectively `false` no matter what you set it to.
 
+`hl.passageSort`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: `startOffset`
+|===
++
+The UH uses the score to select the candidates for the number of snippets defined by `hl.snippets`,
+but the final sorting of these selected candidates is determined by this option.
+The available choices are: `startOffset`, `endOffset`, and `score`.
```

Comment:
Honestly is a user going to ever choose endOffset?  Could we simplify to only startOffset and maybe not even use that geeky internal word; maybe "natural" would be best understood?